### ISSUE TITLE
TextInputLayout not show hint

### ISSUE THREAD

**Issue by ConCaRo** [braintree/android-card-form#23]
I don't know how to add images so I asked a question at StackOverflow.
[http://stackoverflow.com/questions/41637866/textinput-layout-not-show-hint](http://stackoverflow.com/questions/41637866/textinput-layout-not-show-hint)

Thanks for help.

**Comment by ConCaRo** [comment]
I changed to design support library to compile 25.1.0 or 25.0.1 but still the same problem. I checked version "android.support.design.widget.TextInputLayout" in layout, it is 25.1.0.

**Comment by ConCaRo** [comment]
I fixed this problem by  changing 
`compile 'com.android.support:design:[25.0.0,26.0.0)'`
to 
`compile 'com.android.support:design:25.0.1'`
And add to my project as a module .
This problem has the reason: I using `appcompat-v7:25.0.1` but when I use gradle `compile 'com.braintreepayments:card-form:3.0.3'`, the design support library  changed to 25.1.0 and cause that problem. I changed to the same version of appcombat, desgin support librares. 
Thanks for support.
